PIL/Image.py

```diff
@@ -1092,12 +1154,14 @@ def draft(self, mode, size):
         pass
```

Comment:
It means that at one point we checked for a tuple, and broke people on release.

Otoh, if we document it as a tuple, people being typesafe will get it right, and eventually people using list because it works will get linted out as pep8 starts pushing typing.  

I think our current stance is we specify a tuple, but we don't reject a 2list (anymore)